README.md

```diff
@@ -95,12 +95,23 @@ the instructions in [podman.md](podman.md).
 To run the tests, type `./gradlew test` from the `hibernate-reactive` 
 directory.
 
+#### Starting databases automatically with Testcontainers
+
+If you have Docker installed, the container instances will be automatically started
+by running the tests if you select the `-Pdocker` profile like so:
+
+    ./gradlew test -Pdocker
+    
+TIP: To reuse the same containers across multiple runs, set `testcontainers.reuse.enable=true` in
+a file located at `$HOME/.testcontainers.properties` (create the file if it does not exist).
```

Comment:
I'm suprised that the file is located in the $HOME. Isn't it possible to set for the current project?

Reply:
Nope, the reusable container feature was intentionally designed to have a 2-stage enablement:
1) The container object needs to declare it is reusable by doing `.withReuse(true)`
2) The system where the test is running needs to be configured for reuse (via the properties file)
Item (2) is a bit tedious, but it was put in place mainly so that reusable containers don't get left around on CI machines, since it's only a feature that should be used in local environments.

Reply:
ok